webapp/security/views.py

```diff
@@ -258,3 +259,22 @@ def api_create_notice():
     db_session.commit()
 
     return flask.jsonify({"message": "Notice created"}), 201
+
+
+# CVE views
+# ===
+
+match_cves = re.compile(r"cve-\d{4}-\d{4,7}")
+
+
+def cve(cve_id):
+    if cve_id == "cve":
+        return flask.render_template("security/cve_list.html",)
+    elif match_cves.search(cve_id) is not None:
+        return flask.render_template("security/cve_single.html",)
```

Comment:
Could you call it `cve.html` instead?

Reply:
I agree with @tbille here

Reply:
I pulled @steverydz templates, it should be updated now